openerp/models.py

```diff
@@ -5918,6 +5933,10 @@ def recompute(self):
                     recs.browse(ids)._write(dict(vals))
             # mark computed fields as done
             map(recs._recompute_done, fs)
+        # OpenUpgrade start, reset blacklists:
```

Comment:
@StefanRijnhart Good catch. Other calls were due to database updates done during migration. Although code worked in practice, I moved the reset to a more appropriate place (I hope).